include/alpaka/atomic/AtomicGenericSycl.hpp

```diff
@@ -32,7 +32,7 @@ namespace alpaka::experimental
     {
     };
 
-    namespace detail
+    namespace trait::detail
```

Comment:
That should not be in namespace `alpaka::trait::detail` because the content of this namespace is not used to implement traits. Please leave the namespace as `alpaka::detail`. If you made this because otherwise alpaka was picking the wrong detail namespace, consider qualifying with `alpaka::detail::`.